InvenTree/company/templates/company/manufacturer_part.html

```diff
@@ -324,7 +328,9 @@ <h4>{% trans "Parameters" %}</h4>
 
     deleteManufacturerPart({{ part.pk }}, {
         onSuccess: function() {
+            {% if part.manufacturer %}
             window.location.href = "{% url 'company-detail' part.manufacturer.id %}";
+            {% endif %}
```

Comment:
If we delete the manufacturer part, but no manufacturer is specified, where do we redirect the user? Index page?

Reply:
Index